lib/services/pivotal_tracker/pivotal_tracker_feature_and_requirement_mapping_resource.rb

```diff
@@ -43,6 +43,8 @@ def requirement_list(feature)
     end
   end
   
+  # Looks like this code was never fully hooked up. We removed the options to
+  # use "epic-story-task".
```

Comment:
Does that mean the code is never used? Can we remove it?

Reply:
Well it is called up above, and I didn't want to go any further to investigate. All V1 integrations are basically deprecated at this point, so no amount of code changes are worth it. We are more likely to break something than anything else. The next change to this code will likely be to delete it all.

Reply:
Sounds reasonable.